resources/user-agents/browsers/android-browser/android-browser-4-0.json

```diff
@@ -1780,12 +1780,25 @@
         {
           "match": "*Mozilla/5.0 (#PLATFORM#) applewebkit*khtml*like*gecko*) Version/#MAJORVER#.#MINORVER#*Safari*",
           "platforms": [
-            "Android_5_1", "Android_5_0",
-            "Android_4_5", "Android_4_4", "Android_4_3", "Android_4_2", "Android_4_1", "Android_4_0",
             "Android_B_2_3", "Android_2_3", "Android_2_2", "Android_2_1", "Android_2_0",
             "Android_1_6", "Android_1_5", "Android_1_1", "Android_1_0_B", "Android_1_0"
           ]
         },
+        {
+          "match": "*Mozilla/5.0 (#PLATFORM#) applewebkit*khtml*like*gecko*) Version/#MAJORVER#.#MINORVER#*Mobile*Safari*",
+          "platforms": [
+            "Android_5_1", "Android_5_0",
+            "Android_4_5", "Android_4_4", "Android_4_3", "Android_4_2", "Android_4_1", "Android_4_0"
+          ]
+        },
+        {
+          "match": "*Mozilla/5.0 (#PLATFORM#) applewebkit*khtml*like*gecko*) Version/#MAJORVER#.#MINORVER#*Safari*",
+          "device": "general Tablet",
+          "platforms": [
+            "Android_5_1", "Android_5_0",
+            "Android_4_5", "Android_4_4", "Android_4_3", "Android_4_2", "Android_4_1", "Android_4_0"
```

Comment:
Did Android 4.5 really exist?

Reply:
I don't believe so, `4.4.4` was the last official 4.x release (https://en.wikipedia.org/wiki/Android_version_history).

It's possible that `4.5` was a preview/developer release that never officially released and then became what we know as `5.0`.

I'd vote to remove `Android_4_5`. :)

Reply:
Please remove it.

Reply:
removed.